### PR TITLE
Improve flow and fix typos Ch2

### PR DESCRIPTION
**Pull request by SHAESEN2**
I would mention several alternatives to r2RTF as it might not be the preferred choice for many companies. There are some other valid options that are more flexible.

I would make the code very defensive in case of breaking changes to the package. Only show functions that are intended to stay.

### CODE REVIEW

File: tlf-overview.Rmd [hunk 1]
```diff
@@ -4,44 +4,43 @@
...
 
-Within eCTD, clinical study reports (CSR) are located at module 5. 
+Within eCTD, clinical study reports (CSRs) are located at module 5. 
 [ICH E3 guidance](https://www.ich.org/page/efficacy-guidelines) provides 
 compilation of structure and content of clinical study reports.
 
-A typical CSR contain full details of an individual clinical study 
-including clinical and statistical description, presentations, and analyses. 
-A large number of tables and figures are incorporated into the main text and appendices of a CSR. 
+A typical CSR contains full details on the methods and results of a individual clinical study.
```
Comment (elong0527) [inline tlf-overview.Rmd:18]:
```suggestion
A typical CSR contains full details on the methods and results of an individual clinical study.
```

**Comment by elong0527** [comment]
> I would mention several alternatives to r2RTF as it might not be the preferred choice for many companies. There are some other valid options that are more flexible.

Yes, I asked reader to refer Task View at the end of this section. https://elong0527.github.io/r4csr/overview.html#r2rtf

> I would make the code very defensive in case of breaking changes to the package. Only show functions that are intended to stay.

It is a good suggestion. Could you provide an example to improve this?

**Comment by SHAESEN2** [comment]
Hey,

OK. I will have a look at the task view later.
As suggestion for defensiveness:
- I think we need to be aware that many pharmaceutical companies have standards wrt layout of those tables, listings and figures
- The code you provide are for basic manipulations to get the final tbl into rtf without any layout. I think this is good, but we should consider showing wrappers that do all the layout in one go. Wrappers are typically more robust as you can easily change what happens under the hood without the user noticing it

**Comment by elong0527** [comment]
Got it, I created an issue to add additional RTF layout discussion.

**Pull request merged**